crates/volta-core/src/error/details.rs

```diff
@@ -707,10 +707,11 @@ Use `volta pin node` to pin Node first, then pin a Yarn version."
             ),
             ErrorDetails::NoPlatform => write!(
                 f,
-                "Could not determine Node version.
+                "Node is not available.
 
-Use `volta pin node` to select a version for a project.
-Use `volta install node` to select a default version."
+To run any Node command, first set a project or default version.
```

Comment:
I'm finding "set a project or default version" pretty opaque. I think what you're trying to get across is that `volta pin node` will ensure the Node version is available as part of what it accomplishes. But I'm wondering if this is just far enough from what the user is trying to accomplish that it's worth eliminating the advice about `volta pin` altogether here?

IOW, if I'm trying to run Node and it's not available, probably the only real advice I need for now is to run `volta install node`. Selecting a Node version for a project is just kind of not in my critical path, even though it would happen to make a Node version available.

Reply:
That's a good point, if the user just wants to run `node`, we can point them to how to do that, then if they want to learn more advanced stuff (like `volta pin`), we have docs and `volta help` to guide them. That'll also let us reduce this down to a single-line CTA, which is more in line with most of our other errors.